docs/content/specs/shared/_index.md

```diff
@@ -787,6 +787,16 @@ See:
 
 <br>
 
+#### ID: RMFR8 - Category: Composition - Dependency on child and other resources
+
+A resource module **MAY** contain references to other resource modules, however **MUST NOT** contain references to non-AVM modules nor AVM pattern modules.
```

Comment:
Can we add a call out to:

- https://azure.github.io/Azure-Verified-Modules/specs/bicep/#id-bcpfr1---category-composition---cross-referencing-modules
- https://azure.github.io/Azure-Verified-Modules/specs/terraform/#id-tffr1---category-composition---cross-referencing-modules

Reply:
Done